lib/dal/src/migrations/U2404__add_documentation_to_props.sql

```diff
@@ -0,0 +1,64 @@
+-- Add the new columns to the props table.
```

Comment:
Good on you for going the create_v2 route but we could also just get away with replacing the original v1 at this point